packages/ceramic-doctype-tile/src/tile-doctype.ts

```diff
@@ -60,11 +60,20 @@ export class TileDoctype extends Doctype {
      * @param opts - Initialization options
      */
     static async makeGenesis(params: DocParams, context?: Context, opts: DocOpts = {}): Promise<Record<string, any>> {
-        if (!context.did || !context.did.authenticated) {
-            throw new Error('No DID authenticated')
+        const metadata = params.metadata? params.metadata : { owners: [] }
+        const is3id = metadata.tags?.includes('3id')
+        if (is3id) {
+            if (metadata.owners.length === 0) {
+                throw new Error('The owner of the 3ID needs to be specified')
+            }
```

Comment:
I don't think we need any special logic for 3IDs.

Reply:
agree 👍